### ISSUE TITLE
Can't Login Through Okta

### ISSUE THREAD

**Issue by cardonator** [a85/POSTMan-Chrome-Extension#6234]
**Describe the bug**
I am unable to log in to the Postman app or Chrome extension through Okta. The Okta sign in works (badly) in Chrome but it doesn't sign the extension in. Since the extension is deprecated, I also tried in the native app 6.7.1 and 7.0.7 and both have the same behavior.

**To Reproduce**
1. Go to 'Sign In' in the top right
2. Click on 'Enterprise user? Sign in here'
3. Type in the team name
4. Click on Okta SAML as the sign in provider
5. See the "Something went wrong" page with no way to diagnose the problem.

**Expected behavior**
Normally this would allow me to log in and see my team's workspace from inside postman

**App information (please complete the following information):**
 - App Type Chrome App and Native App
 - Postman Version 6.7.1 and 7.0.7
 - OS: Ubuntu 18.04

**Comment by preethammavin** [comment]
Hi @cardonator 
Can you write to us at help@getpostman.com, lets sort this out.

**Comment by preethammavin** [comment]
Hey @cardonator were you able to get this sorted, I am unable to reproduce it

**Comment by preethammavin** [comment]
Closing the issue due to inactivity.

**Comment by cardonator** [comment]
Hi all, sorry for not responding on this earlier. I actually figured out what the issue was but I couldn't ever get the DevTools inside PostMan to tell me what the specific code error the app was getting was.

The issue ended up being that on my corporate internal network they try to use Windows Authentication to automatically log people into our SAML provider. On Linux, this basically pops up a Basic Auth popup which then closes because it's not the Windows Auth provider. That popup for some reason was causing Postman to freak out.

When I tried to log in to Postman outside of the corporate network, it worked just fine.
